flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/TaskExecutorResourceUtils.java

```diff
@@ -688,4 +669,55 @@ public static Configuration adjustConfigurationForLocalExecution(final Configura
 
 		return modifiedConfig;
 	}
+
+	public static Configuration adjustConfigurationForStandaloneSetups(final Configuration configuration) {
+		return getConfigurationMapLegacyTaskManagerHeapSizeToConfigOption(
+			configuration, TaskManagerOptions.TOTAL_FLINK_MEMORY);
+	}
+
+	public static Configuration adjustConfigurationForContainerizedSetups(final Configuration configuration) {
+		return getConfigurationMapLegacyTaskManagerHeapSizeToConfigOption(
+			configuration, TaskManagerOptions.TOTAL_PROCESS_MEMORY);
+	}
```

Comment:
I think this method rather belongs to `ActiveResourceManagerFactory` than to `TaskExecutorResourceUtils`.